app/controllers/api/v1/statuses/reblogs_controller.rb

```diff
@@ -14,12 +14,17 @@ def create
   end
 
   def destroy
-    @status = status_for_destroy.reblog
-    @reblogs_map = { @status.id => false }
+    begin
+      @status = status_for_destroy.reblog
```

Comment:
I don't think this can raise an error. It would just be nil.

Reply:
`status_for_destroy` can